code/game/machinery/computer/pod.dm

```diff
@@ -1,21 +1,38 @@
 /obj/machinery/computer/pod
 	name = "mass driver launch control"
 	desc = "A combined blastdoor and mass driver control unit."
+	/// Connected mass driver
 	var/obj/machinery/mass_driver/connected = null
-	var/title = "Mass Driver Controls"
+	/// ID of the launch control
 	var/id = 1
-	var/timing = 0
+	/// If the launch timer counts down
+	var/timing = FALSE
+	/// Time before auto launch
 	var/time = 30
+	/// Range in which we search for a mass drivers and poddoors nearby
 	var/range = 4
 
-
 /obj/machinery/computer/pod/Initialize()
 	. = ..()
 	for(var/obj/machinery/mass_driver/M in range(range, src))
 		if(M.id == id)
 			connected = M
+			break
 
+/obj/machinery/computer/pod/process()
```

Comment:
```suggestion
/obj/machinery/computer/pod/process(delta_time)
```

Reply:
Processing procs now accept a delta_time value, although if you use the appropriate defines and macros you won't need to utilise this as process() will only be used to check if the cooldown timer has expired and then fire off alert() before returning PROCESS_KILL.